tools/eventlog_metadata.in

```diff
@@ -133,7 +133,8 @@ enum gc_phase : uint16_t {
     "stw_leader",
     "major_finish_sweeping",
     "minor_finalizers_admin",
-    "domain/condition_wait"
+    "domain/condition_wait",
+    "resize minor heap reservation"
```

Comment:
This new field is the only one with the space character `' '` in it. I don't know if this will cause pain elsewhere, but it feels odd not to match the style of the other fields. 
```suggestion
    "resize_minor_heap_reservation"
```

Reply:
Indeed, it should to be `resize_minor_heap_reservation`

Reply:
Actually it should be `domain/resize_minor_heap_reservation`.
Because the event is called `EV_DOMAIN_RESIZE_MINOR_HEAP_RESERVATION`.

Reply:
Thank you! It's done